### PR TITLE
Gravitar, Black Widow, and Space Duel support

### PR DESCRIPTION
**Pull request by mdeslaur**
Here is support for Gravitar, Black Window, and Space Duel.

These have been tested on Gravitar and Space Duel boards.

### CODE REVIEW

File: libraries/C6502Cpu/CGravitarGame.cpp [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+// TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+// EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+#include "CGravitarGame.h"
+
+//
+// Gravitar Set 1
+//
+//                                         01   02   04   08   10   20   40   80  100  200  400  800
+static const UINT16 s_romData2n_1_1[] = {0x16,0x85,0x85,0x85,0x85,0x85,0x7a,0x57,0xab,0x20,0xbb,0x00}; // 136010-101.d1
```
Comment (prswan) [inline libraries/C6502Cpu/CGravitarGame.cpp]: You're adding a lot of image sets here. The way the Arduino compiles these means they end up in data RAM and there is only 8Kb of that. Games that I've been touching I've been moving over to a different way of expressing data2n - see CGalaxianGame.cpp for a space saving format and invocation using a new CGame constructor.

**Review by prswan — COMMENTED** [review]
Nice! Just one suggestion on saving space with PROGMEM based data2n.

File: libraries/C6502Cpu/CGravitarGame.cpp (hunk 1 above)
Comment (mdeslaur) [reply libraries/C6502Cpu/CGravitarGame.cpp]: Oh, cool. Let me fix that up and I'll update the PR.

**Review by prswan — APPROVED** [review]
Thanks!

**Pull request merged**